digital-credentials/support/iframe2.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<body>
+<script>
+async function messageListener(event) {
+  if (event.data != "gain_user_activation") {
+    return;
+  }
+  await test_driver.bless("gain user activation", async function() {
+    console.log(navigator.userActivation.isActive ? "Has User Activation" : "Does not have user activation");
+    return;
+  });
```

Comment:
Can you try:

```suggestion
  await test_driver.bless("gain user activation");
  console.log(navigator.userActivation.isActive ? "Has User Activation" : "Does not have user activation");
```

Reply:
I tested the suggestion and it doesn't make requesting transient user activation for a cross-origin <iframe> which isn't the first on the page work.

Reply:
Maybe:

```suggestion
  test_driver.set_test_context(parent);
  await test_driver.bless("gain user activation", async function() {
    console.log(navigator.userActivation.isActive ? "Has User Activation" : "Does not have user activation");
    return;
  });
```

per https://web-platform-tests.org/writing-tests/testdriver.html#using-test-driver-in-other-browsing-contexts?

Reply:
Thanks for the suggestion! I tried patching your suggestion locally and it does not help the cross-origin iframe get user activation.